graph-partitioning/graph_partitioning.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 D-Wave Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ------ Import necessary packages ----
+import networkx as nx 
+from collections import defaultdict
+from itertools import combinations
+from dwave.system.samplers import DWaveSampler
+from dwave.system.composites import EmbeddingComposite
+
+# ------- Set tunable parameters -------
+num_reads = 1000
+gamma = 156
+
+# ------- Set up our graph -------
+G = nx.gnp_random_graph(40, 0.2)
+
+print("\nGraph on",len(G.nodes),"nodes created with",len(G.edges),"out of",len(G.nodes)*(len(G.nodes)-1)/2,"possible edges.\n")
```

Comment:
Line breaks should happen automatically, so `\n` is not needed.

I think the following might be a more readable than the existing print.
```
print("Graph on {} nodes created with {} out of {} possible edges.".format(len(G.nodes), len(G.edges), len(G.nodes) * (len(G.nodes)-1) / 2))
```